src/modules/app/app.ts

```diff
@@ -185,13 +208,20 @@ export class App {
         window.open(`${window.location.origin}?id=${id}`, '_blank');
     }
 
-    public getSum(): number {
+    public onClickCart(): void {
+        window.location.href = `${window.location.origin}?cart`;
+        this.drawCartPage();
+    }
+
+    public getSum(): number {;
         const array: number[] = this.getSelectedProducts();
         let sum: number = 0;
-        productsData.forEach((item) => {
-            if(array.includes(item.id)) {
-                sum += item.price;
-            }
+        array.forEach((id) => {
```

Comment:
It's really bad idea to name variable `array`.  I'm not even sure that you need a dedicated variable here.

Reply:
Fixed name

Reply:
Fixed.